src/main/java/io/github/cottonmc/resources/CottonResources.java

```diff
@@ -8,13 +8,12 @@
 public class CottonResources implements ModInitializer {
 
 	public static final String MODID = "cotton-resources";
-
-	public static ModLogger logger = new ModLogger(MODID, "COTTON RESOURCES");
-	public static CottonResourcesConfig config = ConfigManager.loadConfig(CottonResourcesConfig.class);
+	public static ModLogger LOGGER = new ModLogger(MODID, "COTTON RESOURCES");
+	public static CottonResourcesConfig CONFIG = ConfigManager.loadConfig(CottonResourcesConfig.class);
```

Comment:
Why are these fields all-caps? They aren't `public static final`.

Reply:
They should be actually, missed that when renaming

Reply:
You should specify a name for CottonResourcesConfig due to the change in how config is loaded; right now this would create a config file without an extension.

Reply:
I've seen ".json5" field in Config lib and it was used

Reply:
Try running the code. I ran into this issue I'm talking about in Epicurean, but that might just have been because I'm using a custom name.

Reply:
Checked, works fine